frame/bridge/ethereum/backing/src/lib.rs

```diff
@@ -28,8 +28,8 @@ pub use weights::WeightInfo;
 
 #[cfg(test)]
 mod mock;
-#[cfg(test)]
-mod test_with_linear_relay;
+// #[cfg(test)]
+// mod test_with_linear_relay;
```

Comment:
linear relay pallet can be deleted.

Reply:
Would like to do some code cleaning in another PR.